sdk/storage/azure-storage-blob/tests/test_common_blob.py

```diff
@@ -1833,6 +1834,17 @@ def test_set_blob_permission(self):
         self.assertEqual(permission.delete, True)
         self.assertEqual(permission.write, True)
         self.assertEqual(permission._str, 'wrdx')
+    
+    def test_transport_closed_only_once(self):
+        transport = RequestsTransport()
+        url = self._get_account_url()
+        credential = self._get_shared_key_credential()
+        blob_name = self._get_blob_reference()
+        with BlobServiceClient(url, credential=credential, transport=transport) as bsc:
+            assert transport.session is not None
+            with bsc.get_blob_client(self.container_name, blob_name) as bc:
+                assert transport.session is not None
+            assert transport.session is not None  # Right now it's None
```

Comment:
"right now it's none" as we expect it to fail?  (In which case skip?)  Or some other meaning?

Reply:
It meant it was none before this change..ill remove it.. :)